app/models/list.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class List < ApplicationRecord
+  COLORS = %w[danger success dark light primary secondary warning info].freeze
+  PRIORITIES = %w[High Medium Low].freeze
```

Comment:
good job using freeze.

Reply:
Thanks, but rubocop was the one who told me this